### ISSUE TITLE
[TS]: Separate interface for render options

### ISSUE THREAD

**Issue by Dangoo** [claus/storyblok-rich-text-react-renderer#12]
Thanks for bringing TS support (#4) with #9! It allowed us to remove our custom typings from the project :)
But would it be possible to separate the interfaces of `render` arguments, so one can create the render options object outside of the function call?

**Is:**
```typescript
render(richtextData, {
  [NODE_PARAGRAPH]: (children) => (
    <Paragraph>{children}</Paragraph>
  ),
  ...
})
```

**Desired outcome:**
```typescript
import {render, RenderOptions} from 'storyblok-rich-text-react-renderer'

const richtextOptions: RenderOptions = {
  [NODE_PARAGRAPH]: (children) => (
    <Paragraph>{children}</Paragraph>
  ),
  ...
}

render(richtextData, richtextOptions)
```

Let me know what you think or if I can help in any way, cheers!

**Comment by claus** [comment]
@Dangoo Yeah that sounds like a good idea. Do you by chance have that locally already and could contribute? Or maybe @TGlide could take a look? Otherwise i can also look into that.

**Comment by TGlide** [comment]
I can take a look into that! I'll take a look on monday

**Comment by claus** [comment]
Awesome, thank you @TGlide !

**Comment by claus** [comment]
Thanks so much, @TGlide!
@Dangoo can you take a quick look at [#13](https://github.com/claus/storyblok-rich-text-react-renderer/pull/13)? Does that solve your issue?

**Comment by Dangoo** [comment]
@claus sure thing, just did :)

**Comment by TGlide** [comment]
Thanks for the comments! I'll take a look at the suggetions tomorrow, but at first glance they seem like good ideas IMO

**Comment by claus** [comment]
Published as 2.5.2
Thanks guys, i appreciate your help on this!
